app/controllers/visits_controller.rb

```diff
@@ -18,30 +18,43 @@
 # INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR
 # TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-class Dashboard::VisitsController < Dashboard::BaseController
+class VisitsController < ApplicationController
   respond_to :json, :js, :html
 
+  before_action :initialize_service_request
+  before_action :authorize_identity
+
   def edit
-    @visit = Visit.find(params[:id])
-    @portal = params[:portal]
+    @visit  = Visit.find(params[:id])
+    @admin  = params[:admin]
+    @tab    = 'billing_strategy'
+    @page   = params[:page]
 
     respond_to do |format|
       format.js
     end
   end
 
-  # Used for x-editable update and validations
   def update
```

Comment:
Replacement for ServiceCalendarsController#update. Update a visit group and load instance variables to rerender partials.